docs/source/predictors/latest.rst

```diff
@@ -13,9 +13,8 @@ algorithm but rather approximates the latest software stack resolution.
 The randomness introduced causes incompatibilities with pip/Pipenv/Poetry
 resolver.
 
-Check `termial random
-<https://medium.com/@fridex/termial-random-for-prioritized-picking-an-item-from-a-list-a65a4f563224>`__
-and `its optimized form
-<https://medium.com/@fridex/optimizing-termial-random-by-removing-binomial-coefficient-e39b9ca7aaa3>`__
-which is used to prioritize picking more recent releases considering version
-specifier.
+..  Check `termial random
+  <https://medium.com/@fridex/termial-random-for-prioritized-picking-an-item-from-a-list-a65a4f563224>`__
```

Comment:
This one was moved to https://dev.to/fridex/termial-random-for-prioritized-picking-an-item-from-a-list-22jh

Reply:
I have update the first link with the web-address provided